.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+    - 1.7
+install:
+    - cd cluster-state-service; make get-deps; cd ../
+    - cd daemon-scheduler; make get-deps; cd ../
+script:
+    - cd cluster-state-service; make; make release; cd ../
```

Comment:
I don't think `release` is the right target here. It has a dependency on `docker`. `unit-tests` would be the right one I think, also I think you should `&&` instead of `;` so that the exit code is captured correctly.